components/Header/Header.jsx

```diff
@@ -12,9 +21,6 @@ import { BiLogInCircle } from "react-icons/bi";
 
 import { FaWhmcs } from "react-icons/fa";
 
-import { RiCloseLine, RiYoutubeFill, RiGithubFill, RiTwitterFill, RiLinkedinFill } from "react-icons/ri";
-
-
 import { Container } from "reactstrap";
 import { useSession, signOut, signIn } from "next-auth/react";
```

Comment:
Move this import statement up to line 2

Reply:
I have written import statements on top of component only. So I think it's not creating any issue.